src/Gallery/index.tsx

```diff
@@ -32,21 +34,20 @@ export const Gallery: FC<{ onUse?: (image: GalleryItem) => void }> = ({ onUse })
 
     const tags = splitKeywords(searchQueryTags) || [];
     const type = settingsPanel.activeTab as GalleryFileType;
-
     const search = api.gallery
       .search({ domainId: domain._id, page: 1, size: 10, type, tags, filter })
       .then(({ items }: { items: GalleryItem[] }) => setItems(items))
       .catch(error => setError(error.message || error));
 
     return toast.promise(search, {
-      error: 'Image upload failed 🤯',
+      error: 'Image search failed 🤯',
```

Comment:
I guess this search could be for more than just images. Could we make it "Gallery search failed"?